Libs/DICOM/Widgets/ctkDICOMObjectListWidget.cpp

```diff
@@ -131,16 +131,18 @@ ctkDICOMObjectListWidgetPrivate::~ctkDICOMObjectListWidgetPrivate()
 void ctkDICOMObjectListWidgetPrivate::setFilterExpressionInModel(qRecursiveTreeProxyFilter* filterModel, const QString& expr)
 {
   const QString regexpPrefix("regexp:");
+  QString regexPattern;
   if (expr.startsWith(regexpPrefix))
   {
     filterModel->setFilterCaseSensitivity(Qt::CaseSensitive);
-    filterModel->setFilterRegularExpression(expr.right(expr.length() - regexpPrefix.length()));
+    regexPattern = expr.mid(regexpPrefix.length());
   }
   else
   {
     filterModel->setFilterCaseSensitivity(Qt::CaseInsensitive);
-    filterModel->setFilterWildcard(expr);
+    regexPattern = QRegularExpression::escape(expr);
```

Comment:
Wildcard is better for most users (? and * can replace one or more characters). They don't know regular expressions. You may prevent special characters to be interpreted as regex by escaping, but that does not bring back the wildcard functionality.

Would it be possible to fix filtering by wildcard?